src/ENVPAK.CPP

```diff
@@ -111,6 +144,7 @@ WStr enExeInfo(		// retrieve build date/time, linker version, etc from exe
 	codeSize = 0;
 	WStr linkerVersion;
 
+	#if !defined( ELIMINATE_WINDOWS)
```

Comment:
@nealkruis What would you like me to do with this section of code. Leave it like it is?

Reply:
`if _WIN32` for now.

Reply:
Ok. I'll get rid off all the ELIMINATE_WINDOWS macros and request you when I am done.